ui/app/AppLayouts/Profile/controls/AccountDelegate.qml

```diff
@@ -0,0 +1,22 @@
+import StatusQ.Components 0.1
```

Comment:
Can we rename the file to WalletAccountDelgate?

Reply:
done